### ISSUE TITLE
#-1344299508------------- ARIA toggle fields must have an accessible name

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#464]
<strong>Source:</strong>`<div aria-checked="false" aria-label="" class="toggle" role="checkbox" tabindex="0"><svg aria-hidden="true" class="check-svg" viewBox="0 0 16 16"><path d=""></path></svg><slot></slot></div>`
<strong>Element Location: </strong>["calcite-checkbox[data-test=\"header-selection-checkbox\"]",".toggle[role=\"checkbox\"][aria-label=\"\"]"]
<strong>Impact: </strong>serious
<strong>failureSummary:</strong> Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attributeTags: [Ljava.lang.String;@44aeae34
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
